content/runbooks/alertmanager/AlertmanagerFailedReload.md

```diff
@@ -0,0 +1,33 @@
+---
+title: AlertmanagerFailedReload
```

Comment:
Do you want spaces on the title or just the alert name match? The other PR has the spaces

Reply:
this is the only documentation in alert manager component that existed. I did not try to do something different. I believe this is used for the menu and help for google to find the doc.

https://github.com/swissquote/runbooks/blob/main/content/runbooks/alertmanager/AlertmanagerMembersInconsistent.md
and here is the template they provide : https://github.com/prometheus-operator/runbooks/blob/main/content/docs/add-runbook.md

I don't know what the weight is for.